conda_forge_tick/cli.py

```diff
@@ -209,6 +237,62 @@ def make_import_to_package_mapping(
     import_to_pkg.main(ctx, max_artifacts)
 
 
+@main.command(name="parse-feedstock")
+@feedstock_option
+@existing_feedstock_metadata_option
+def parse_feedstock(
+    feedstock: str,
+    existing_feedstock_metadata: str,
+) -> None:
+    """
+    Parse a feedstock and update the feedstock metadata.
+    """
+    from . import feedstock_parser
+
+    check_feedstock_param(feedstock, existing_feedstock_metadata)
+
+    feedstock_parser.main_feedstock_parser(
+        name=feedstock,
+        node_attrs_file=existing_feedstock_metadata,
+    )
+
+
+@main.command(name="find-latest-feedstock-version")
+@feedstock_option
+@existing_feedstock_metadata_option
+def find_latest_feedstock_version(
+    feedstock: str,
+    existing_feedstock_metadata: str,
+) -> None:
+    """
+    Find the most recent version for a feedstock.
```

Comment:
Like above, maybe we can add one or two sentences more that explain what the output of this command is and what files are changed on disk when running it?